fgd/point/momentum/momentum_df_weaponspawner.fgd

```diff
@@ -0,0 +1,15 @@
+@PointClass base(BaseEntityPoint)
+	appliesto(MOMENTUM) 
+	= momentum_df_weaponspawner : "Defrag weapon spawner."
+	[
+	weapon_slot(choices) : "Weapon" : 2 : "The weapon to spawn." =
```

Comment:
We don't use it mainly because I'm not 100% on its use but there is a mode for FGD compilation that replicates what's in engine more exactly and wouldn't want to break it here: For choices like this, they have an accompanying engine version, eg

`weapon_slot[engine](integer) : "Weapon" : 2`